ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/ptp.yml

```diff
@@ -40,10 +40,11 @@ ptp:
 # port-channels
 port_channel_interfaces:
   - name: Port-Channel5
-    type: switched
     description: DC1_L2LEAF1_Po1
-    vlans: 110,201
-    mode: trunk
+    switchport:
```

Comment:
Add `enabled: true` on the tests where needed, to avoid changes to the output.

Reply:
Fixed